scripts/variantstore/docs/aou/AOU_DELIVERABLES.md

```diff
@@ -99,9 +99,15 @@ The Callset Stats and S&P files can be simply `gsutil cp`ed to the AoU delivery
 ## Running the VAT pipeline
 To create a BigQuery table of variant annotations, you may follow the instructions here:
 [process to create variant annotations table](../../variant_annotations_table/README.md)
-
 The pipeline takes in the VDS and outputs a variant annotations table in BigQuery.
 
+Once the VAT table is created and a tsv is exported, the AoU research workbench team should be notified of its creation and permission should be granted so that several members of the team have view permission.
+
+- Grant `BigQuery Data Viewer` permission to specific people's PMI-OPS accounts. This will include members of the AoU research workbench team.
+- Copy the bgzipped, tarred export of the VAT into the pre-delivery bucket.
```

Comment:
Is this `gs://prod-drc-broad` (confusingly also referred to in this same document as the "delivery bucket")? Also any suggested path for the VAT tarball or is that something for the Google Doc below?

Reply:
I've put that all in the linked Google Doc.